src/lib/swapper/swappers/ThorchainSwapper/utils/getLimit/getLimit.ts

```diff
@@ -113,7 +115,7 @@ export const getLimit = async ({
       // Else the return fee is the outbound fee of the sell asset's chain
       default: {
         const sellAssetTradeFeeCryptoHuman = fromBaseUnit(
-          bnOrZero(sellAssetAddressData?.outbound_fee),
+          bnOrZero(sellAssetAddressData.unwrap().outbound_fee),
```

Comment:
Do we know `sellAssetAddressData` didn't error?

Reply:
Absolutely not - 7dd148e3902f2e6f45a7b836067eeb0b89746830